src/derivers/bip32.ts

```diff
@@ -276,14 +284,14 @@ function generatePublicKey({
   curve,
 }: GeneratePublicKeyArgs): DerivedKeys {
   const entropy = hmac(sha512, chainCode, publicExtension);
-  const keyMaterial = entropy.slice(0, 32);
-  const childChainCode = entropy.slice(32);
+  const keyMaterial = entropy.subarray(0, 32);
```

Comment:
Are subarrays safe for this?

Reply:
It's not used anymore after it's returned from this function, so in theory it should be fine. If we want to be on the safe side, we could return a copy of the original byte array in the actual `SLIP10Node` class, but that might be overkill?

Reply:
Nah, let's create fresh copies in the `SLIP10Node` class. Seems like a bad idea to have byte data with lifecycles persisting across multiple functions/contexts.

Reply:
Alternatively, since we're working with pretty tiny amounts of memory, we could just use `.slice()` everywhere. I don't know that we'll see a significant performance difference either way.